sources/packages/web/src/views/aest/institution/Profile.vue

```diff
@@ -1,9 +1,13 @@
 <template>
   <v-card class="mt-4">
     <div class="mx-5 py-4">
-      <div class="mb-2">
-        <p class="category-header-large color-blue">Profile</p>
-      </div>
+      <body-header title="Profile" class="m-1">
+        <template #actions>
+          <v-btn text :color="COLOR_BLUE" @click="editProfile"
+            ><font-awesome-icon :icon="['fas', 'cog']" class="mr-2" />Edit
```

Comment:
i think its regular icon in figma,
![image](https://user-images.githubusercontent.com/77353155/158478828-5933e50f-89b4-4936-bfed-a775007f8191.png)

Reply:
Yes but regular is a PRO icon. 
![image](https://user-images.githubusercontent.com/54600590/158502613-2d314b46-7111-440a-915d-be5e71375e06.png)